nifty-core/src/main/java/com/facebook/nifty/ssl/SSLDefaults.java

```diff
@@ -0,0 +1,27 @@
+package com.facebook.nifty.ssl;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class SSLDefaults {
+    public static final List<String> SERVER_DEFAULTS;
+
+    static {
+        SERVER_DEFAULTS = new ArrayList<String>();
```

Comment:
will change to use ImmutableList. This is just a good list in general which uses good ciphers suites which are forward secure and also efficient. This is created from experience. This would not normally be updated